.github/CHANGELOG.md

```diff
@@ -28,25 +28,26 @@
   ```
 
 * Keyword arguments are supported inside operations; for example,
-  `MeasureHomodyne(select=[0, 1])` or `MeasureFock(dark_counts=[1, 0.5, 3])`
+  `MeasureHomodyne(select=[0, 1])` or `MeasureFock(dark_counts=[1, 0.5, 3])`.
   [(#22)](https://github.com/XanaduAI/blackbird/pull/23)
 
 * Indexing is supported inside both modes and arguments; for example,
   `MZgate(list[2], list[0]) | [list[3], list[1]]))`. All indices allow
   expressions such as `phases[-3 + 2*2]`.
   [(#22)](https://github.com/XanaduAI/blackbird/pull/23)
 
-<h3>Improvements</h3>
-
-<h3>Breaking changes</h3>
-
 <h3>Bug fixes</h3>
 
-* Templates are no longer changed when calling them and replacing the parameters with actual values.
+* Templates are no longer changed when calling them and replacing the parameters
+  with actual values.
```

Comment:
Seems that this would be relevant somewhere too: https://github.com/XanaduAI/blackbird/pull/33

Reply:
#33 should probably not be here at all, since it was added and removed without ever being released with it. It was practically a mistake which was removed prior to this release. :slightly_smiling_face:

Reply:
Oh, I see! Could you leave a small note on it in the PR description? (And not sure, maybe a tag like `[Reverted]` in the PR name could help?) Not super urgent could just help later if we'd like to go through the PRs

Reply:
I think this sounds like a nice idea. Is this something that we usually do in SF and PL?

Reply:
I think it's fine to leave it off (assuming it has been fully reverted). For a user reading the changelog, it will not be a relevant entry, since it is not providing any details about the changes between this and prior releases.

For the same reason we do not include changelog entries for fixing bugs on un-released features.